.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
Can we add the bug label automatically to this?

Reply:
Yes, we can. But should we maybe do it like in scikit-learn and also add a label "needs triage" so that we know that it hasn't been checked yet?